grid_map_cv/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ find_package(sensor_msgs REQUIRED)
 
 find_package(OpenCV REQUIRED
   COMPONENTS
-    opencv_photo
```

Comment:
Interesting... It seems both component names are supported and resolve. Is there a reason to switch?

Reply:
Searching in github, components without `opencv_` prefix seem majority such as `core`, `highgui`, `improc`.

Reply:
I took a deep dive and found this is consistent with the intent of the config file that's exported.